### PR TITLE
feat Enable fee calculation for logion.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#123

### CODE REVIEW

File: calc/pkg/package.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
   "files": [
     "calc_bg.wasm",
     "calc.js",
-    "calc_bg.js",
```
Comment (gdethier) [inline calc/pkg/package.json:16]: Why?

File: src/chains-config/logionControllers.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { ControllerConfig } from '../types/chains-config';
+
+/**
+ * Controllers that Sidecar will always default to. This will always be
+ * the optimal controller selection for Polkadot and Kusama.
+ */
```
Comment (gdethier) [inline src/chains-config/logionControllers.ts:5]: Above comment is incorrect.

File: src/chains-config/logionControllers.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		'AccountsVestingInfo',
+		'Blocks',
+		'BlocksExtrinsics',
+		'NodeNetwork',
+		'NodeTransactionPool',
+		'NodeVersion',
+		'PalletsAssets',
+		'PalletsStakingProgress',
+		'PalletsStorage',
+		'Paras',
```
Comment (gdethier) [inline src/chains-config/logionControllers.ts]: Above controller is probably useless currently, but that's a detail.

**Review by gdethier — APPROVED** [review]
Congratulations!

File: calc/pkg/package.json (hunk 1 above)
Comment (benoitdevos) [reply calc/pkg/package.json:16]: That was auto-fixed by yarn or my ide. That file just does not exist.

File: src/chains-config/logionControllers.ts (hunk 3 above)
Comment (benoitdevos) [reply src/chains-config/logionControllers.ts]: Removed.

File: src/chains-config/logionControllers.ts (hunk 2 above)
Comment (benoitdevos) [reply src/chains-config/logionControllers.ts:5]: Comment updated.

**Pull request merged**